src/Chapter2.hs

```diff
@@ -741,7 +752,9 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int->[[Int]] -> [[Int]]
+contains _ []=[]
+contains y (x:xs) = if elem y x then [x] ++ contains y xs else []++ contains y xs
```

Comment:
You can use `filter` function here instead of the pattern matching, but this is nice solution already (you'll just need to replace `++` with `:` operator 🙂 )